src/api/test/functional/group_test.rb

```diff
@@ -43,7 +43,10 @@ def test_get_group
   end
 
   def test_create_modify_and_delete_group
-    xml = '<group><title>new_group</title></group>'
+    FactoryBot.create(:confirmed_user, login: 'hans')
+    FactoryBot.create(:confirmed_user, login: 'luke')
+
+    xml = '<group><title>new_group</title><maintainer userid="hans"/><person><person userid="luke"/></person></group>'
```

Comment:
Why is this necessary now?

Reply:
Because now the validation is done against the schema, and the schema requires a `maintainer` and a `person` elements present.

Reply:
Alright...